src/summon/browzine-summon-adapter.js

```diff
@@ -174,7 +174,7 @@ browzine.summon = (function() {
 
   function showDirectToPDFLink() {
     var enableShowDirectToPDFLink = false;
-    var config = browzine.articlePDFDownloadLinkEnabled;
+    var config = browzine.summonArticlePDFDownloadLinkEnabled || browzine.articlePDFDownloadLinkEnabled;
```

Comment:
How about we have the platform-specific variable consistently either on the left or right side of the `||` ?  That way, we can clearly state which item takes precedence if anyone has some funny configuration.

Reply:
Awww, Hey @karlbecker! Just remembered why I didn't do that. The nonprefix was the original, so we have those out there. Then prefix was the second high-level iteration. Returning to the non-prefix should probably take the precedence going forward.

It's just this particular check is a weird case. Let's see if the new tests reveal anything.